src/open_inwoner/conf/locale/nl/LC_MESSAGES/django.po

```diff
@@ -6431,6 +6431,10 @@ msgstr "U dient in ieder geval 1 contactpersoon toe te voegen voor een plan."
 msgid "This field is required when not using a template"
 msgstr "Dit veld is vereist."
 
+#: open_inwoner/plans/forms.py:93
+msgid "The end date of the plan cannot precede the end dates of the actions in the selected template."
+msgstr "De einddatum van het plan kan niet voorafgaan aan de einddatums van de acties in het geselecteerde sjabloon."
+
```

Comment:
The word "voorafgaan" is not allowed for language accessibility reasons: https://ishetb1.nl/search?word=voorafgaan

So change this to a bit more simplified Dutch: 
"Het plan kan niet eerder eindigen dan de einddatums van de acties in het geselecteerde sjabloon."